dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESMappingAPIImpl.java

```diff
@@ -216,215 +216,215 @@ public String toJson(Contentlet con) throws DotMappingException {
+				if(contentMeta.exists() && contentMeta.length()>0) {
 
-                    String contentData=APILocator.getFileAssetAPI().getContentMetadataAsString(contentMeta);
+					String contentData=APILocator.getFileAssetAPI().getContentMetadataAsString(contentMeta);
 
-                    String lvar=con.getStructure().getVelocityVarName().toLowerCase();
+					String lvar=con.getStructure().getVelocityVarName().toLowerCase();
 
-                    mlowered.put(lvar+".metadata.content", contentData);
-                }
-            }
+					mlowered.put(lvar+".metadata.content", contentData);
+				}
+			}
 
 			//The url is now stored under the identifier for html pages, so we need to index that also.
 			if(con.getStructure().getStructureType() == Structure.STRUCTURE_TYPE_HTMLPAGE){
 				mlowered.put(con.getStructure().getVelocityVarName().toLowerCase() + ".url", ident.getAssetName());
 				mlowered.put(con.getStructure().getVelocityVarName().toLowerCase() + ".url_dotraw", ident.getAssetName());
 			}
 
-            return mlowered;
+			return mlowered;
 		} catch (Exception e) {
 			//Logger.error(this.getClass(), e.getMessage(), e);
 			throw new DotMappingException(e.getMessage(), e);
 		}
 	}
 
-    /**
-     * Set the Vanity Url URI in the content map
-     * @param con The current vanity url Contentlet
-     * @param contentletMap Contentlet properties map
-     * @param ident Contentlet
-     */
-    private void populateVanityUrlPath(Contentlet con, Map<String, String> contentletMap, Identifier ident) {
-        String vanityUrlPath;
-        try{
-            VanityUrl vanityUrl = APILocator.getVanityUrlAPI().getVanityUrlFromContentlet(con);
-            vanityUrlPath = VanityUrlUtil.fixURI(vanityUrl.getURI());
-            if(vanityUrlPath != null){
-                contentletMap.put(ESMappingConstants.VANITY_URL,vanityUrlPath);
-            }
-        }catch(Exception e){
-            Logger.warn(this.getClass(), "Cannot get Vanity URL for contentlet.id : " + ((ident != null) ? ident.getId() : con) + " , reason: "+e.getMessage());
-            throw new DotRuntimeException(e);
-        }
-    }
-
-
-    public Object toMappedObj(Contentlet con) throws DotMappingException {
+	/**
+	 * Set the Vanity Url URI in the content map
+	 * @param con The current vanity url Contentlet
+	 * @param contentletMap Contentlet properties map
+	 * @param ident Contentlet
+	 */
+	private void populateVanityUrlPath(Contentlet con, Map<String, String> contentletMap, Identifier ident) {
+		String vanityUrlPath;
+		try{
+			VanityUrl vanityUrl = APILocator.getVanityUrlAPI().getVanityUrlFromContentlet(con);
+			vanityUrlPath = VanityUrlUtil.fixURI(vanityUrl.getURI());
+			if(vanityUrlPath != null){
+				contentletMap.put(ESMappingConstants.VANITY_URL,vanityUrlPath);
+			}
+		}catch(Exception e){
+			Logger.warn(this.getClass(), "Cannot get Vanity URL for contentlet.id : " + ((ident != null) ? ident.getId() : con) + " , reason: "+e.getMessage());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 151 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)